src/main/java/me/earth/earthhack/impl/modules/client/hud/HUD.java

```diff
@@ -240,6 +244,30 @@ protected void renderModules() {
             final String dir = RotationUtil.getDirection4D(false);
             renderText(dir, 2, height - 3 - RENDERER.getStringHeightI() * 2 - animationY);
         }
+
+        if (totems.getValue()) {
+            ScaledResolution sr = new ScaledResolution(Minecraft.getMinecraft());
+            RenderItem itemRender = mc.getRenderItem();
+            int width = sr.getScaledWidth();
+            int height = sr.getScaledHeight();
+            int totems = mc.player.inventory.mainInventory.stream().filter(itemStack -> itemStack.getItem() == Items.TOTEM_OF_UNDYING).mapToInt(ItemStack::getCount).sum();
+
+            if (mc.player.getHeldItemOffhand().getItem() == Items.TOTEM_OF_UNDYING) {
+                totems += mc.player.getHeldItemOffhand().getCount();
+            }
+
```

Comment:
Please use `InventoryUtil.getCount(item)` instead all of this.

Reply:
I did not know about this ty for the info!